interface/frontend/src/components/detect-and-select/CandidateList.vue

```diff
@@ -103,7 +103,6 @@
               noduleCentroid.x = x
               noduleCentroid.y = y
               noduleCentroid.z = z
-              console.log(`coordinates saved, updating client model`, x, y)
```

Comment:
Were these added in this PR or elsewhere? If the former, why not remove them from your commits?  :)

Reply:
I have removed this commit and updated two commits that introduced these changes.